README.md

```diff
@@ -49,18 +49,18 @@ struct Settings {
     #[miniconf(defer)]
     array_defer: [i32; 2],
     // ... or deferring to two levels (index and then inner field name)
-    #[miniconf(defer)]
-    array_miniconf: miniconf::Array<Inner, 2>,
+    #[miniconf(defer(2))]
```

Comment:
WOuld the term `depth` make more sense than `defer` now? Both technically work, just a thought.